tests/quantization/bnb/test_mixed_int8.py

```diff
@@ -134,14 +134,16 @@ def test_get_keys_to_not_convert(self):
         from transformers.integrations.bitsandbytes import get_keys_to_not_convert
 
         model_id = "mosaicml/mpt-7b"
-        config = AutoConfig.from_pretrained(
-            model_id, trust_remote_code=True, revision="72e5f594ce36f9cabfa2a9fd8f58b491eb467ee7"
-        )
-        with init_empty_weights():
-            model = AutoModelForCausalLM.from_config(
-                config, trust_remote_code=True, code_revision="72e5f594ce36f9cabfa2a9fd8f58b491eb467ee7"
-            )
-        self.assertEqual(get_keys_to_not_convert(model), ["transformer.wte"])
+        # TODO: to revert once: https://huggingface.co/mosaicml/mpt-7b/discussions/83 is fixed
+        # config = AutoConfig.from_pretrained(
+        #     model_id, trust_remote_code=True, revision="72e5f594ce36f9cabfa2a9fd8f58b491eb467ee7"
+        # )
+        # with init_empty_weights():
+        #     model = AutoModelForCausalLM.from_config(
+        #         config, trust_remote_code=True, code_revision="72e5f594ce36f9cabfa2a9fd8f58b491eb467ee7"
+        #     )
+        # self.assertEqual(get_keys_to_not_convert(model), ["transformer.wte"])
```

Comment:
This should be split into a separate test e.g. `test_get_keys _to_not_convert_remote_code` and explicitly skipped with `unittest.skip` as well as an issue raised in this repo to track whether it's been resolved.